tb.go

```diff
@@ -0,0 +1,36 @@
+package merkletree
+
+import "github.com/coniks-sys/libmerkleprefixtree-go/crypto"
+
+type TemporaryBinding struct {
+	index []byte
+	value []byte
+	sig   []byte
+}
+
+func NewTemporaryBinding(pad *PAD, key string, value []byte) *TemporaryBinding {
+	str := pad.currentSTR
+	index := computePrivateIndex(key)
+	tb := str.SerializeWithSignature()
```

Comment:
I see your confusion now. I went back to look at Certificate Transparency's [Signed Tree Head spec](https://tools.ietf.org/html/rfc6962#section-3.5), after which STRs are modeled, and I think technically, the STR should be only the signature. 

What this means for ``prevStrHash`` and also ``TB`` (which yes, the notation should really use concatenation for consistency, though I believe commas are more proper crypto notation), is that these should only use ``STR.signature`` in their respective computations.

However, when TBs and STRs are sent to the client (and eventually to auditors in STRs' case), we will still have to include all the values to allow for signature verification.